bitnami/solr/values.yaml

```diff
@@ -0,0 +1,387 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## Add labels to all the deployed resources
+##
+commonLabels: {}
+
+## Add annotations to all the deployed resources
+##
+commonAnnotations: {}
+
+## Kubernetes Cluster Domain
+##
+clusterDomain: cluster.local
+
+## Extra objects to deploy (value evaluated as a template)
+##
+extraDeploy: []
+
+## Number of nodes
+##
+replicaCount: 1
+
+service:
+  type: LoadBalancer
+  # HTTP Port
+  port: 8983
+  # HTTPS Port
+  httpsPort: 8983
```

Comment:
I don't think it makes sense to have the HTTPS port be the same as the HTTP port. In any case, I don't think you are actually using this value in the chart, so you can remove it.